packages/components/src/Banner/Banner.tsx

```diff
@@ -103,17 +89,12 @@ export function Banner({
       </div>
 
       {dismissible && (
-        <button
-          type="button"
-          className={styles.closeButton}
-          onClick={handleClose}
-          aria-label="Close this notification"
-        >
-          <Icon
-            name="cross"
-            color={JOBBER_RETHEME ? "interactiveSubtle" : iconColors[type]}
+        <div className={styles.closeButton}>
+          <ButtonDismiss
+            ariaLabel={"Dismiss notification"}
```

Comment:
Just comment that this will break some JO tests but not that big of an issue

Reply:
Honestly I'm happy to revert the ariaLabel change if it speeds things up; it's really unrelated to the retheme. Although if it's just a matter of updating the strings in related Jobber tests, I can live with that 😄

Reply:
Yeah, it is just updating strings. Super quick and not many failed when I ran it on JO. Most of the tests were snapshots. I like the update